app/scripts/lib/multichain-api/scope/transform.ts

```diff
@@ -38,15 +38,9 @@ export const flattenScope = (
 };
 
 export const mergeScopeObject = (
-  // scopeStringA: CaipChainId,
   scopeObjectA: ScopeObject,
-  // scopeStringB: CaipChainId,
   scopeObjectB: ScopeObject,
 ) => {
-  // if (scopeStringA !== scopeStringB) {
-  //   throw new Error('cannot merge ScopeObjects for different ScopeStrings')
-  // }
-
   // TODO: Should we be verifying that these scopeStrings are flattened / the scopeObjects do not contain `scopes` array?
```

Comment:
It looks like we do this in the `mergeScopes` function? Can this TODO be removed as well?

Reply:
i removed this comment in a separate open PR, but it won't hurt to remove it in this one too, and so we shall 🙏